_posts/news/2016-01-25-announcing-scalajs-0.6.6.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title: Announcing Scala.js 0.6.6
+category: news
+tags: [releases]
+permalink: /news/2016/01/25/announcing-scalajs-0.6.6/
+---
+
+
+We are excited to announce the release of Scala.js 0.6.6!
+
+It has been a long while since the last release, almost 5 months already.
+That was too long, and we apologize.
+In the future, we hope never to let more than 2 months elapse between consecutive releases.
+
+Besides bug fixes, this release brings several major improvements:
+
+* `js.TupleN`, a JS equivalent of Scala tuples
+* Support for JUnit
+* Better support of constructors of `@ScalaJSDefined` classes:
+  they can now have overloads, default parameters and varargs
+* A completely redesigned internal API for the linker, whose main visible impact should be reduced memory usage and improved speed
+* `js.ConstructorTag[C]`, a `ClassTag` equivalent to `js.constructorOf[C]`
+
+<!--more-->
+
+## Getting started
+
+If you are new to Scala.js, head over to
+[the tutorial]({{ BASE_PATH }}/tutorial/).
+
+## Release notes
+
+As a minor release, 0.6.6 is (almost, see Breaking changes below) backward source and binary compatible with previous releases in the 0.6.x series.
+Libraries compiled with earlier versions can be used with 0.6.6 without change.
+However, it is not forward compatible: libraries compiled with 0.6.6 cannot be used by projects using 0.6.{0-5}.
+
+Please report any issues [on GitHub](https://github.com/scala-js/scala-js/issues).
+
+## Breaking changes
```

Comment:
Maybe add to this section that anything in `org.scalajs.tools` might have experienced a breaking change.